packages/react/src/components/Typography/Typography.stories.mdx

```diff
@@ -50,3 +52,37 @@ function Demo() {
   );
 }`}
 />
+
+## Variants
+
+The `variant` prop gives access to a pre-defined scale of typographic values defined in the theme. These values include
+various heading levels (h1, h2, h3, etc.) as well as body text levels (body1, body2, etc) and can be used to apply
+consistent typography throughout your application.
+
+Additionally, you can also use the `variant` prop to control the font size, weight, line height, and other typographic
+properties.
+
+export const SampleText = 'Lorem ipsum.';
+
+<Canvas>
+  <Story name="Variants">
+    {Object.entries(defaultTheme.typography)
+      .sort((a, b) => {
+        return b[1].fontSize?.split('rem')[0] - a[1].fontSize?.split('rem')[0];
+      })
+      .map(([key, value]) => {
+        const isVariant = key !== 'button' && (value.hasOwnProperty('fontSize') || value.hasOwnProperty('fontWeight'));
+        if (!isVariant) {
+          return null;
+        }
+        console.log(JSON.stringify(defaultTheme.typography));
```

Comment:
Shall we remove this?

Reply:
Fixed with https://github.com/wso2/oxygen-ui/pull/40/commits/1ac617fbe10c03596f88e8c98c2dc105d7cea938